packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -83,7 +117,7 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, Dr
     fontWeight: 'bold',
   }),
 
-  toggleIndicator: ({ variables: v }): ICSSInJSStyle => ({
+  toggleIndicator: ({ props: p, variables: v }): ICSSInJSStyle => ({
```

Comment:
Why we need props here?

Reply:
we don't, I forgot it there while testing something, removed, thanks!